### PR TITLE
Black formatter GitHub action

### PR DESCRIPTION
**Pull request by fgypas**
- Black formatter GitHub action for lint checks. 
- Formatted code with black
- I used line width of 127.

### CODE REVIEW

**Review by CaibinSh — APPROVED** [review]

**Comment by CaibinSh** [comment]
Unbelievable that class object with '()' worked, I though this was right! Thanks @fgypas

**Comment by fgypas** [comment]
> Unbelievable that class object with '()' worked, I though this was right! Thanks @fgypas

It's automatic the formatting though black.

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 ![machine learning](https://img.shields.io/badge/machine_learning-005AF0?style=for-the-badge.svg)
 ![variational autoencoders](https://img.shields.io/badge/variational_autoencoders-005AF0?style=for-the-badge.svg)
 ![denoising](https://img.shields.io/badge/denoising-005AF0?style=for-the-badge.svg)
+<br>
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
```
Comment (fgypas) [inline README.md:9]: Addition of black badge

File: .github/workflows/black.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          options: "--check --verbose --line-length 127"
+          src: "./scAR"
```
Comment (fgypas) [inline .github/workflows/black.yml:13]: Checks if the code has been formatted with black. Locally to format the code you need to run:
`black --line-length 127 scAR`

**Review by CaibinSh — APPROVED** [review]

**Pull request closed without merging**